src/gpio_chip.h

```diff
@@ -44,7 +45,7 @@ struct gpio_chip_event_handle {
  * Struct for event data
  */
 struct gpio_chip_event_data {
-    int timestamp;
+    __u64 timestamp;
```

Comment:
```suggestion
    uint64_t timestamp;
```

You might need to include `stdint.h` and you probably don't need `linux/types.h` any more. The type should be the same, but I like using the stdint.h versions over all of the other varieties for consistency.